pkg/controller/kfservice/kfservice_controller_suite_test.go

```diff
@@ -38,15 +38,16 @@ var cfg *rest.Config
 
 func TestMain(m *testing.M) {
 	t := &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join("..", "..", "..", "config", "crds")},
+		CRDDirectoryPaths: []string{filepath.Join("..", "..", "..", "config", "crds"),
+			filepath.Join("..", "..", "..", "test", "crds")},
```

Comment:
what's going on here?

Reply:
Added the configuration/route crd files which needs to be installed for mock apiserver for test to work. I put in test directory instead of config directory because otherwise it gets installed for real deployment. Waiting https://github.com/kubernetes-sigs/controller-runtime/issues/394 to be resolved then we can remove these files and directly point to the crd files in knative vendor directory.